packages/jest-haste-map/src/crawlers/node.ts

```diff
@@ -31,7 +31,14 @@ async function hasNativeFindSupport(
 
   try {
     await which('find');
```

Comment:
do we still need the `which` now, or can we remove it?

Reply:
I think we can remove it. I'll update.

Reply:
Should be enough to essentially just revert #9579